### PR TITLE
add support alert styling to Text Block

### LINKED ISSUE #263: Defect: Critical - Body Builder Support page - Body Builder Support page is not correctly configured / has different blocks
The Body Builder Support page on hlx.page ( https://main--vg-volvotrucks-us--hlxsites.hlx.page/parts-and-services/services/body-builder-support/ ) is not correctly configured and has different / missing blocks compared to https://www.volvotrucks.us/parts-and-services/services/body-builder-support/

**Steps to reproduce:**
01. Go to https://main--vg-volvotrucks-us--hlxsites.hlx.page/parts-and-services/services/body-builder-support/
02. Go to https://www.volvotrucks.us/parts-and-services/services/body-builder-support/
03. Notice how differently the pages are configured

**Expected result**
Body Builder Support page should be correctly configured

**Actual result**
Body Builder Support page is not correctly configured

**NOTE:** it is unclear to me if this defect is related with this item that is currently open at the time of logging this ticket block: body builder news #234 , if that is the case then this ticket can be closed as invalid

### PR DESCRIPTION
**Pull request by dkuntze**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #263 

Test URLs:
- Before: https://main--vg-volvotrucks-us--hlxsites.hlx.page/parts-and-services/services/body-builder-support/
- After: https://263-bb-support--vg-volvotrucks-us--hlxsites.hlx.page/parts-and-services/services/body-builder-support/

### CODE REVIEW

File: blocks/text/text.css [hunk 1]
```diff
@@ -36,3 +36,24 @@ div.text.black {
...
+    display: flex;
+}
+
+div.text.alert.block img {
+    height: auto;
+    max-width: 40px;
+}
+
+div.text.alert.block > div:first-of-type {
+    float: left;
```
Comment (Buuhuu) [inline blocks/text/text.css:50]: the float should not be necessary as the cells are positioned within the flex box

File: blocks/text/text.css [hunk 2]
```diff
@@ -36,3 +36,24 @@ div.text.black {
...
+    padding-left: 70px;
+    padding-right: 40px;
+    padding-top: 20px;
```
Comment (Buuhuu) [inline blocks/text/text.css:53]: Now that the headings don't have a top margin anymore it is probably better to set the padding top on the `div.text.alert.block` to have it consistent over the full width of the block.

**Review by Buuhuu — CHANGES_REQUESTED** [review]

**Comment by synox** [comment]
I would rather move away from `Text` and add this variant to `Columns`?

**Comment by Buuhuu** [comment]
> I would rather move away from Text and add this variant to Columns?
move away from `Text` in general or for this use case? I don' think that the specifics of Columns fit here:

-  the columns of the Columns block are usually equally sized. here we have a narrow (shrink) column with the image and a wide column with the text (grow)  
- Columns stack vertically on mobile, the alert icon does not stack at all.

**Comment by synox** [comment]
I mean to move away from Text in general. Based on: 

> It is definitely an anti-pattern to have things that are represented natively as default content and put them into a block, so something like a Text, Heading or Image block yields a bad authoring perspective.
https://www.hlx.live/docs/davidsmodel

**Comment by Buuhuu** [comment]
I agree and disagree with that. 

> anti-pattern to have things that are represented natively as default content and put them into a block

One example: small text cannot be represented natively (well it can but the font sizes are ignored). So we need a block for this as we have some disclaimers and other texts with a smaller font size. If we call it "disclaimer" or "text (s)" depends on what works for the authors. so far they were used to use headings to take control on the font size and style. 

Wrt to the alert, we could have an Alert block that does the same thing, right.

**Comment by synox** [comment]
Thanks Dirk, these are valid points. Feel free to ignore my comments 😅.

File: blocks/text/text.css (hunk 1 above)
Comment (dkuntze) [reply blocks/text/text.css]: done

**Review by synox — APPROVED** [review]

**Pull request merged**